aws.yaml

```diff
@@ -1,4 +1,101 @@
 - active: false
+  apps:
+    - app: chart-operator
+      version: 0.11.2
+      use-upgrade-force: true
+      changelogs:
+        - description: Remove CPU Limits
+          type: Changed
+          urls:
+            - https://github.com/giantswarm/chart-operator/pull/335
+    - app: coredns
+      version: 1.1.3
+      componentVersion: 1.6.5
+      use-upgrade-force: false
+      changelogs:
+        - description: Fix HPA manifests for Kubernetes 1.16.
+          type: Changed
+          urls:
+          - https://github.com/giantswarm/coredns-app/blob/master/CHANGELOG.md#v113-2020-01-08
+    - app: cert-exporter
+      version: 1.2.1
+      use-upgrade-force: true
+      changelogs:
+        - description:
+          type: Changed
+          urls:
+            - https://github.com/giantswarm/cert-exporter/blob/master/CHANGELOG.md#121-2019-12-24
+    - app: kube-state-metrics
+      version: 1.0.0
+      componentVersion: 1.9.0
+      use-upgrade-force: true
+      changelogs:
+        - description: Upgraded to kube-state-metrics new release 1.9.0
+          type: Changed
```

Comment:
I kept this type as it is but honestly, I don't think this is necessary since the description is already there, and most of them are Changed.

Reply:
That's true. Changelogs would be messy and not sure it should be together with version information.

Reply:
We need to keep `type` its part of the changelog format and we do use different values. e.g when adding new apps.

https://keepachangelog.com/en/1.0.0/